echo_server.py

```diff
@@ -21,10 +24,49 @@ def echo_server():
                 if len(response) < 32:
                     done = True
                 text_received = "{}{}".format(text_received, response)
-            conn.sendall(text_received)
+            if text_received:
+                response_user = parase_request(text_received)
```

Comment:
I can tell by the way this is used that you aren't following the spec from the assignment.